integration/core/test_cli.py

```diff
@@ -106,18 +106,18 @@ def getNow():
     return datetime.datetime.utcnow().replace(microsecond=0).isoformat()
 
 
-def open_replica(grpc_client):
+def open_replica(grpc_client, size):
```

Comment:
There are 2 `open_replica` (one in common, and the other in core package) . Should we consolidate and keep one only?

Reply:
> A few comments about testing, and suggest having the default value for `open_replica` or others w/ a new size arg, so you don't need to modify the test functions. WDYT?
> 
> ```python
> open_replica(grpc_replica2)
> ```

Yes, good idea.

Reply:
Merged the two functions.